ethereum/api/src/test/java/org/hyperledger/besu/ethereum/api/jsonrpc/websocket/methods/EthSubscribeIntegrationTest.java

```diff
@@ -62,7 +62,7 @@ public void before() {
   public void shouldAddConnectionToMap(final TestContext context) {
     final Async async = context.async();
 
-    final JsonRpcRequest subscribeRequest = createEthSubscribeRequest(CONNECTION_ID_1);
+    final JsonRpcRequestBody subscribeRequest = createEthSubscribeRequest(CONNECTION_ID_1);
```

Comment:
rename createEthSubscribeRequest to createEthSubscribeRequestBody

Reply:
done